apiserver/apiserver_test.go

```diff
@@ -73,9 +78,22 @@ func (s *apiserverConfigFixture) SetUpTest(c *gc.C) {
 	s.tlsConfig.Certificates = []tls.Certificate{*coretesting.ServerTLSCert}
 	s.mux = apiserverhttp.NewMux()
 
+	modelCache, err := modelcache.NewWorker(modelcache.Config{
+		Logger:               loggo.GetLogger("test"),
+		StatePool:            s.StatePool,
+		PrometheusRegisterer: noopRegisterer{},
+		Cleanup:              func() {},
+	})
+	s.AddCleanup(func(c *gc.C) { workertest.CleanKill(c, modelCache) })
+	c.Assert(err, jc.ErrorIsNil)
+	var controller *cache.Controller
+	err = modelcache.OutputFunc(modelCache, &controller)
```

Comment:
This is now `modelcache.ExtractCacheController` I believe.

Reply:
Oops, didn't catch that one.